src/explore-education-statistics-admin/src/services/releaseDataFileService.ts

```diff
@@ -101,7 +101,7 @@ const releaseDataFileService = {
     data.append('metaFile', request.metadataFile);
 
     const file = await client.post<DataFileInfo>(
-      `/release/${releaseId}/data?name=${request.name}`,
+      `/release/${releaseId}/data?name=${encodeURIComponent(request.name)}`,
```

Comment:
I don't think we need to actually do this directly, You can pass a third `options` param to `client.post`, which can contain some `params` (query parameters). Really we should be doing this in lots of other places as well.

Reply:
done - cheers